clients/python/rostam/_vecwire.py

```
@@ -308,3 +870,194 @@ def decode_get_result(body: bytes) -> Optional[Dict[str, Any]]:
+    p = off + 3
+    for _ in range(missing_count):
+        missing.append(struct.unpack(">H", body[p:p + 2])[0])
+        p += 2
+    return degraded, missing, trailer_end
+
+
+def _decode_docs_raw(body: bytes, off: int = 0) -> Tuple[List[Dict[str, Any]], int]:
+    """Mirrors ops.frameVectorDocsN + the typed unmarshal: one EncodeVectorDocs
+    block at body[off:]. Wire per doc: [id:u64][distance:f32][score:f32]
+    [contentLen:u32][content][metaLen:u32][metaJSON]. Returns (docs, next_off)."""
+    (count,) = struct.unpack(">I", body[off:off + 4]); off += 4
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)